lib/__tests__/helpers/createMockManager.js

```diff
@@ -7,6 +7,7 @@ module.exports = function() {
     openMenu: jest.fn(),
     handleKeyDown: jest.fn(),
     handleClick: jest.fn(),
+    handleBlur: jest.fn(),
```

Comment:
preface: I'm new here-

I suspect that this is why `npm test` fails.

Snapshots expected `{undefined}`, and in reality the `onBlur` prop was not being added at all, and so the test would fail. It seems reasonable to either add a mock for `handleBlur` and expect `[Function]` in the snapshots, or else remove the `onBlur={undefined}` from each of the snapshots.